README.md

```diff
@@ -95,7 +94,7 @@ You can read more about **new Date()** [here](https://developer.mozilla.org/en-U
 > https://discord.gg/EvbMshU
 
 ### Sources: 
-> https://www.worldometers.info/coronavirus/ 
+> https://www.worldometers.info/coronavirus/  
```

Comment:
Need to add 1 more line here to separate 2 lines of quotes

Reply:
was added in #193

Reply:
@buster95 I've removed this change. ✌️